Security/TwoFactor/Provider/Totp/TotpAuthenticatorInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Scheb\TwoFactorBundle\Security\TwoFactor\Provider\Totp;
+
+use OTPHP\TOTP;
+use Scheb\TwoFactorBundle\Model\Totp\TwoFactorInterface;
+
+interface TotpAuthenticatorInterface
+{
+    public function checkCode(TwoFactorInterface $user, string $code): bool;
```

Comment:
OK. I usually don't add any PHPDoc when the method behaviour is obvious.
But I can add them with a simple description.

Reply:
Great! Just want to keep it consistent. The other interfaces (since they're integration points) are also documented a bit more than just plain method signatures :)